content/components/relative-time.mdx

```diff
@@ -30,7 +30,7 @@ This component does not include any styling. It formats a timestamp as a localiz
 
 #### Past dates
 
-For past dates, the relative time component outputs different formats depending on the time difference. Within a month's time, the output shows a relative time[^1] in the format of `X seconds/minutes/hours/days ago`. If the time difference is less than 1 minute, it shows `just now`. If the date is in past months, the date time[^2] format `on D MMM` is used. If the date is in the past years, the format `on D MMM, YYYY` is used.
```

Comment:
Probably worth giving this a proper name such as "Long Format". The format is localised so it may be `on D MMM, YYYY` on some users' machines while on others it may be `on MMM D, YYYY`, or even `on YYYY MMM D`

Reply:
Just updated the future and past relative dates with the "long format" description and examples